Cargo.toml

```diff
@@ -151,14 +153,14 @@ debug = true
 # ALWAYS document what PR the commit hash is part of, or when it was merged into the upstream trunk.
 
 # Temporary patch until next egui release
-ecolor = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
-eframe = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
-egui-wgpu = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
-egui-winit = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
-egui = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
-egui_extras = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
-emath = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
-epaint = { git = "https://github.com/emilk/egui", rev = "dd417cfc1acdf7ac88f214015d4d8e7a21835d30" }
+ecolor = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
+eframe = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
+egui-wgpu = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
+egui-winit = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
+egui = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
+egui_extras = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
+emath = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
+epaint = { git = "https://github.com/emilk/egui", rev = "a08f179c" }
```

Comment:
My new favourite commit is 3c4223c6b18dc2ed441fc5b0077117e614565067 :)

Reply:
I'd prefer b896d641c578c04603adf07a9350b8bfc3c7ed9f though :smile:

Reply:
Maybe we can reach an agreement on that 🤝

Reply:
actually no, we can't. Need an even newer one